### PR TITLE
persist: replace incorrect MB name with correct MiB in cfg constant

### PR DESCRIPTION
**Pull request by danhhz**
### Motivation

   * This PR refactors existing code.

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

File: src/persist-client/src/cfg.rs [hunk 1]
```diff
@@ -221,11 +221,12 @@ impl PersistConfig {
     }
 }
 
-pub(crate) const MB: usize = 1024 * 1024;
+#[allow(non_upper_case_globals)]
+pub(crate) const MiB: usize = 1024 * 1024;
```
Comment (danhhz) [inline src/persist-client/src/cfg.rs:225]: I originally typed this up as MIB because of the lint, but I think it's less clear that way. wdyt?

Comment (guswynn) [reply src/persist-client/src/cfg.rs:225]: I like this more!

**Review by guswynn — APPROVED** [review]
nice

**Comment by danhhz** [comment]
TFTR!

**Pull request merged**